CMakeLists.txt

```diff
@@ -287,6 +291,12 @@ if (DOWNLOAD_OBJECTS)
             file(REMOVE \$ENV{DESTDIR}/${CMAKE_INSTALL_FULL_DATADIR}/${PROJECT}/object/objects.zip)\n\
         endif ()")
 endif ()
+if (DOWNLOAD_REPLAYS)
+    set(REPLAY_DIR "\$ENV{DESTDIR}${CMAKE_INSTALL_FULL_DATADIR}/${PROJECT}/testdata/replays")
```

Comment:
This should follow the structure of other such entries – allow caching of downloaded files.